README.md

```diff
@@ -21,6 +21,31 @@ You can deploy Che plugin registry on Openshift with command.
              -p IMAGE_TAG="latest" \
              -p PULL_POLICY="IfNotPresent"
 ```
+
+## Kubernetes
+
+You can deploy Che plugin registry on Kubernetes using [helm](https://docs.helm.sh/).
+
+```bash
+
+NAMESPACE="kube-che"
+DOMAIN="$(minikube ip).nip.io"
```

Comment:
here it looks like more for minikube than k8s

Reply:
I have updated the sentence above to make it clear that in the example below the target platform is minikube.